fossunited/patches/v1_0/migrate_only_flags_to_allowed_session_types.py

```
@@ -0,0 +1,48 @@
+
+        allowed = "Workshop" if cfp.only_workshops else NON_WORKSHOP_TYPES
+
+        try:
+            frappe.db.set_value(
+                EVENT_CFP,
+                cfp.name,
+                "allowed_session_types",
+                allowed,
+                update_modified=False,
+            )
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)